app/controllers/application_controller.rb

```diff
@@ -11,7 +11,8 @@ def cascade_block(block_path)
 
   def cascade_format(format_path)
     # Make data definition a local within partial as variable current_page
-    render_to_string(partial: format_path, locals: {current_page: @data_definition})
+    render_to_string(partial: format_path, locals: {current_page: @data_definition,
+                                                    contentRoot: @data_definition.document})
```

Comment:
`current_page` should be dropped here, but it's used by some of the other `format` partials.